### PR TITLE
Added variance declarations for Stream methods

### PR DESCRIPTION
**Pull request by karlicoss**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ before_script:
   - emulator -avd test -no-skin -no-audio -no-window &
   - android-wait-for-emulator
   # unlock screen:
-  - adb shell input keyevent 82 &
+  - adb shell input keyevent 82
```
Comment (karlicoss) [inline .travis.yml]: I think this is more correct, since unlocking screen should be synchronous

**Comment by konmik** [comment]
Hi, is it important for you to merge it asap? I'm on vacation right now, the next weekend will be a more convenient time to do it for me.

Thanks for the contribution, BTW :)

**Comment by karlicoss** [comment]
@konmik nope, there's no rush, it's ok. Have a nice vacation :)

**Comment by karlicoss** [comment]
Oh, almost forgot: this kinda breaks interface compatibility in case someone using this library extended `Stream` and overridden one of these methods (which is quite unlikely), but it is easily fixed just by using same appropriate types. Worth mentioning in release notes though.

**Comment by konmik** [comment]
Do you know how to make travis build third-party pull requests? It normally can build only pull requests of users who have write access to the repository.

**Comment by karlicoss** [comment]
@konmik I think that might have been some glitch, travis should build all PRs regardless write access. I bumped by commit and it triggered the build :)

**Comment by konmik** [comment]
I've just released this update as 3.0.0 to not break apis.

Thanks for your contributions! :)

**Pull request merged**